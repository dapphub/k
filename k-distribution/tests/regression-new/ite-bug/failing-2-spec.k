module FAILING-2-SPEC
    imports ITE-BUG
    rule <k> JUMPI 2 bool2Word(X <Int 0) => . </k>
         <pc> 0 => 2 </pc>
endmodule
