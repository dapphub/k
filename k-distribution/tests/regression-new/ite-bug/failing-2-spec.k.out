<generatedTop>
  <k>
    .
  </k>
  <pc>
    1
  </pc>
</generatedTop>
